shared/naturalcrit/markdown.js

```diff
@@ -46,6 +46,27 @@ const mathParser = new MathParser({
 		array       : false, fndef       : false
 	}
 });
+// Add absolute function
+mathParser.functions.abs = function (a) {
+	return Math.abs(a);
+};
```

Comment:
Don't need to define `abs`. It already exists in our `mathParser`, but has just been disabled (set to `false` above). Instead just set to `true`, and move it up with the other `true` functions. Might have to rearrange the block of disabled functions here to make it look nice again.

Reply:
I believe I created the function specifically because I was having difficulty getting the built in version to function. I'll take another swing at it; if there IS an issue with activating the functions, then it's probably better to find and resolve it now.

Reply:
Oh! I figured it out. It was the variable name! Once I changed the variable name to a letter that did not appear in the function name, the issue disappeared.

![image](https://github.com/user-attachments/assets/d4c3bbef-f613-4d0e-aaaf-351bd11c6b1c)

This might be an issue deserving of further investigation, but it falls outside the realm of this PR.